benchmark-integration/jetty-perftest/src/main/java/io/opentelemetry/perftest/Worker.java

```diff
@@ -1,17 +1,20 @@
 package io.opentelemetry.perftest;
 
-import static io.opentelemetry.auto.instrumentation.api.AgentTracer.activeSpan;
-
+import io.opentelemetry.OpenTelemetry;
 import io.opentelemetry.auto.api.Trace;
-import io.opentelemetry.auto.instrumentation.api.AgentSpan;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
 import java.util.concurrent.TimeUnit;
 
 public class Worker {
 
+  private static final Tracer TRACER =
+      OpenTelemetry.getTracerFactory().get("io.opentelemetry.auto");
+
   @Trace
```

Comment:
should you delete this?

Reply:
can't delete it yet (still need to wrap the method in manual start/end span before it can be removed), that's coming in a PR in the near future :-)

Reply:
ack